log/term/colorlogger.go

```diff
@@ -0,0 +1,149 @@
+package term
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"sync"
+
+	"github.com/go-kit/kit/log"
+)
+
+// Color is the abstract color, the zero value is the Default.
+type Color uint8
+
+const (
+	NoColor = Color(iota)
+	Red
+	Green
+	Yellow
+	Blue
+	Magenta
+	Cyan
+	White
+	Default
+
+	maxColor
+)
+
+var resetColorBytes = []byte("\x1b[0m")
+var fgColorBytes [][]byte
+var bgColorBytes [][]byte
+
+func init() {
+	for color := NoColor; color < maxColor; color++ {
+		fgColorBytes = append(fgColorBytes, []byte(fmt.Sprintf("\x1b[%dm", 30+color)))
+		bgColorBytes = append(bgColorBytes, []byte(fmt.Sprintf("\x1b[%dm", 40+color)))
+	}
+}
```

Comment:
Two copies of the package causes problems when trying to interchange values between the public API of the two instances or when interacting with external resources. These arrays have neither problem. Another copy of go-kit (under a different import path) can get at them than no more than any other package.